recipes/ganon/meta.yaml

```diff
@@ -42,10 +49,16 @@ test:
 
 about:
   home: https://github.com/pirovc/ganon
+  summary: "ganon: a read classification tool using Interleaved Bloom Filters"
+  description: |
+          ganon is a k-mer based read classification tool which uses Interleaved Bloom Filters
+          in conjunction with a taxonomic clustering and a k-mer counting-filtering scheme
   license: MIT License
-  summary: 'ganon is a k-mer based read classification tool which uses Interleaved Bloom Filters in conjunction with a taxonomic clustering and a k-mer counting-filtering scheme'
   license_family: MIT
```

Comment:
Can you add `license_file: LICENSE` so the license get's distributed with the package?

Reply:
I did so but circleCI cannot find the file even though it's there and I'm not really sure why

Reply:
Very odd. I'll click the "approve" button and we can revert that change